server/lib/subscriptions.js

```diff
@@ -49,6 +52,14 @@ function hasReachedQuantity(order) {
   return order.Subscription.chargeNumber !== null && order.Subscription.chargeNumber === order.Subscription.quantity;
 }
 
+function getOrderTierPlanName(order) {
```

Comment:
It's too specific, you should just make it return a Boolean and name it `isHostPlan`

Reply:
If doing that you can even move this utility to `lib/plans`

Reply:
I need the name returned, not just a boolean check.

Reply:
You don't need a method to read the name, you can just read it. Give it a try.